cli/bpmetadata/parser/state_parser.go

```diff
@@ -0,0 +1,49 @@
+package parser
+
+import (
+	"encoding/json"
+
+	"github.com/golang/protobuf/jsonpb"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+// ParseOutputTypesFromState parses the terraform.tfstate file and extracts the output types
+func ParseOutputTypesFromState(stateData []byte) (map[string]*structpb.Value, error) { // Change return type
+	var state struct {
```

Comment:
Let's use https://pkg.go.dev/github.com/hashicorp/terraform-json#State to parse out state rather than maintaining our own structs